SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -93,32 +91,44 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 
 	UE_LOG(LogSpatialOSNetDriver, Log, TEXT("Loaded Map %s. Connecting to SpatialOS."), *LoadedWorld->GetName());
 
-	checkf(!SpatialOSInstance->IsConnected(), TEXT("SpatialOS should not be connected already. This is probably because we attempted to travel to a different level, which current isn't supported."));
+	//checkf(!SpatialOSInstance->IsConnected(), TEXT("SpatialOS should not be connected already. This is probably because we attempted to travel to a different level, which current isn't supported."));
```

Comment:
Remove this check. @joshuahuburn perhaps do that in your client travel PR if relevant.